packages/devtools_app/lib/src/screens/memory/panes/chart/chart_pane.dart

```diff
@@ -216,46 +210,35 @@ class _MemoryChartPaneState extends State<MemoryChartPane>
     }
   }
 
-  void _refreshCharts() {
-    // Remove history of all plotted data in all charts.
-    widget.chartControllers.event.reset();
-    widget.chartControllers.vm.reset();
-    widget.chartControllers.android.reset();
-
-    _recomputeChartData();
-  }
-
-  /// Recompute (attach data to the chart) for either live or offline data source.
-  void _recomputeChartData() {
-    widget.chartControllers.event.setupData();
-    widget.chartControllers.event.dirty = true;
-    widget.chartControllers.vm.setupData();
-    widget.chartControllers.vm.dirty = true;
-    widget.chartControllers.android.setupData();
-    widget.chartControllers.android.dirty = true;
-  }
-
   @override
   Widget build(BuildContext context) {
-    return Column(
-      children: [
-        const SizedBox(height: denseRowSpacing),
-        SizedBox(
-          height: scaleByFontFactor(70),
-          child: events.MemoryEventsPane(widget.chartControllers.event),
-        ),
-        SizedBox(
-          child: vm.MemoryVMChart(widget.chartControllers.vm),
-        ),
-        controller.isAndroidChartVisible
-            ? SizedBox(
-                height: defaultChartHeight,
-                child:
-                    android.MemoryAndroidChart(widget.chartControllers.android),
-              )
-            : const SizedBox(),
-        const SizedBox(width: defaultSpacing),
-      ],
+    // TODO(terry): Can Flutter's focus system be used instead of listening to keyboard?
```

Comment:
not for addressing in this PR, but @elliette did some work in the Debugger page to make our key press handling more robust. That could be a good place to look for an example when addressing focus issues in the memory page in the future.